src/dtflw/lazy_notebook.py

```diff
@@ -82,14 +85,25 @@ def input(self, name: str, file_path: str = None, source_table: str = None) -> L
                 self.rel_path
             )
 
-        elif not self.ctx.storage.is_abs_path(input_file_path):
+        elif not self.ctx.storage.is_abs_path(input_file_path) and not input_file_path=="":
             # Bind the input to a given specific file
             input_file_path = self.ctx.storage.get_abs_path(input_file_path)
 
-        self.__inputs[name] = InputTable(
+        input_table = InputTable(
```

Comment:
let's combine these two statements into one. We can calc input_file_path and then create InputTable object

Reply:
@User1000 Done!